<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Load Parameter -->
  <param name="use_cpu"           value="false" />
  
  <!-- Start yolov8 and ros wrapper -->
  <node pkg="yolov8_ros" type="yolo_v8.py" name="yolov8_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov8_ros)/weights/gjs.pt"/>
    
    <!--param name="image_topic"       value="/camera/color/image_raw" /-->    
    <!--param name="image_topic"       value="/usb_cam/image_raw" /-->
    <param name="image_topic"       value="/camera/image_raw" /> 
    
    <param name="pub_topic"         value="/object_position" />
    <!--param name="camera_frame"       value="camera_color_frame"/-->
    <param name="camera_frame"      value="usb_cam"/>
    <param name="visualize"         value="false"/>
    <param name="conf"              value="0.1" />
  </node>
</launch>


